game/script-topics.rpy

```diff
@@ -4406,6 +4406,11 @@ label talk_custom_music_explanation:
     n 1ullss "If you don't know how to check,{w=0.1} then just look at the letters after the period in the file name."
     n 1unmss "You should also be able to see those in the file {i}properties{/i} if they don't appear on the screen at first."
     n 1flrbg "Like I said -{w=0.1} this thing isn't {i}exactly{/i} super modern,{w=0.1} so it won't work with any fancy newer formats,{w=0.1} or weird old ones."
+    n 1uwdaj "Oh,{w=0.75}{nw}"
+    extend 1nlrpu " and if you gotta convert it first,{w=1}{nw}"
+    extend 1nsqpo " don't just rename it."
+    n 1fcsbg "Use a proper converter!"
```

Comment:
Next line uses `extend`, but you have no `w` or `nw` tags here, maybe a mistake?

Reply:
Good catch - thanks!